CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 6.33.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_memcache_instance` ([#22477](https://github.com/hashicorp/terraform-provider-google/pull/22477))
+* **New Resource:** `google_bigtable_logical_view` ([#22499](https://github.com/hashicorp/terraform-provider-google/pull/22499))
+* **New Resource:** `google_bigtable_materialized_view` ([#22475](https://github.com/hashicorp/terraform-provider-google/pull/22475))
+* **New Resource:** `google_compute_region_security_policy_rule` (ga) ([#22443](https://github.com/hashicorp/terraform-provider-google/pull/22443))
+* **New Resource:** `google_compute_region_security_policy` (ga) ([#22443](https://github.com/hashicorp/terraform-provider-google/pull/22443))
+* **New Resource:** `google_os_config_v2_policy_orchestrator_for_folder` ([#22441](https://github.com/hashicorp/terraform-provider-google/pull/22441))
+
+IMPROVEMENTS:
+* beyondcorp: added `upstreams` fields to `google_beyondcorp_application` resource ([#22514](https://github.com/hashicorp/terraform-provider-google/pull/22514))
+* compute: added fields like `raw_key`, `rsa_encrypted_key`, `kms_key_service_account` to all relevant resources on `google_compute_instance_template` and `google_compute_region_instance_template` ([#22503](https://github.com/hashicorp/terraform-provider-google/pull/22503))
+* compute: added `disk_id` to `google_compute_region_disk` resource ([#22462](https://github.com/hashicorp/terraform-provider-google/pull/22462))
+* compute: marked `location` field as required in `google_compute_interconnect` resource ([#22480](https://github.com/hashicorp/terraform-provider-google/pull/22480))
```

Comment:
is this a breaking change? Usually we mark location as computed so it inherits provider level defaults?

Reply:
i guess not breaking since it optional right now but technically required